daemon-scheduler/pkg/deployment/environment.go

```diff
@@ -44,9 +44,10 @@ type Environment interface {
 	// FilterEnvironments returns a list of all environments that match the filters
 	FilterEnvironments(ctx context.Context, filterKey string, filterVal string) ([]types.Environment, error)
 
-	// AddDeployment adds a deployment to the environment if a deployment with
+	//TODO: reconsider how these methods are structured when adding transactions (do we need 2 separate methods? etc)
+	// AddPendingDeployment adds a deployment to the environment if a deployment with
 	// the provided ID does not exist
-	AddDeployment(ctx context.Context, environment types.Environment, deployment types.Deployment) (*types.Environment, error)
+	AddPendingDeployment(ctx context.Context, environment types.Environment, deployment types.Deployment) (*types.Environment, error)
```

Comment:
I'm a little confused. I thought we were moving deployment related functions to deployment.go? Is that not happening anymore or will it be coming in furture CRs?